backend/src/utils/cyper_import_data_from_csv.py

```diff
@@ -1,7 +1,21 @@
 import_data_from_csv_script = """
 LOAD CSV WITH HEADERS FROM 'file:///esg_poc.csv' AS row
 
-MERGE (f:Fund {Name: row.`Fund Name`, Size: toFloat(row.`Fund Size (Billion USD)`)})
+MERGE (f:Fund {Name: row.`Fund Name`, Size: toFloat(row.`Fund Size (Billion USD)`), SizeUnit:
+    CASE
```

Comment:
Tiny point, but could we move the SizeUnit: to the next line, to match the style of Currency?